organizing/speakers-guide.md

```diff
@@ -1,9 +1,62 @@
-This is a guide for speakers in Pyladies Berlin meetups.
+# Speaker guidelines
+This is a guide for speakers at Pyladies Berlin meetups. The following guidelines apply to **talks and workshops**; for workshops, you find additional recommendations below.
 
-TODO add the contents of https://docs.google.com/document/d/1uAOtT6MQYjlKtuJSM6Uz-UPKKk29Ms05rIVVouQghc4/edit?usp=sharing
+* **Audience level:** Expect all levels. From people that have never tried Python before to Python data science professionals and engineers. In our meetups, we aim that beginners (having programmed a bit before) are able to follow most of the things. 
 
-keep common parts on top, if it is too much maybe we can reduce the info to useful level
+* **Coaching:** We offer coaching. That means you can discuss your talk/workshop material with one of us and ask to go through the slides/workshop outline and give feeback. It is also possible to arrange a talk rehearsal session live or via video call. Pleasse get in touch with us!
 
-* Talk 
+#Maybe we could add some guidelines here on talk length, language and ideas for talks or talks that have been given in the past. I couldn't find this in the google doc.
```

Comment:
We can encourage a 20-30 mins that is also suitable for a conference later, but anything less is absolutely welcome as well, also lightning talks of 5 mins. The only thing is that we need to know that in timely manner to set the schedule accordingly.